internal/ui/promts.go

```diff
@@ -0,0 +1,57 @@
+package ui
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type ValidateFn func(input string) error
```

Comment:
docs on exported types please - also `StringPrompt` and `YesNoPrompt`

Reply:
Updated.